workflow/snakemake_rules/export_for_nextstrain.smk

```diff
@@ -439,9 +439,12 @@ def send_slack_message(message, broadcast=False):
     except NoSuchEntryError:
         pass
 
-    response = requests.post("https://slack.com/api/chat.postMessage", headers=headers, data=json.dumps(data))
-    response.raise_for_status()
-    storage.store_if_not_present("slack_thread_ts", response.json()["ts"])
+    try:
+        response = requests.post("https://slack.com/api/chat.postMessage", headers=headers, data=json.dumps(data))
+        response.raise_for_status()
+        storage.store_if_not_present("slack_thread_ts", response.json()["ts"])
+    except Exception as error:
+        print("An error occurred when sending Slack message:", error)
```

Comment:
A full traceback is probably more helpful than just the stringification of the error (which is unfortunately often very terse/uninformative).

```suggestion
        print("An error occurred when sending Slack message:", file=sys.stderr)
        traceback.print_exc()
        print("Oh well. Ignoring and moving on…", file=sys.stderr)
```

(Needs an import of `traceback` and maybe `sys` at the top.)

Reply:
Updated in bf64d625eb9ffc9987816c674685bded53211d1b